website/docs/api/plugins/eslint-plugin.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 6
+id: eslint-plugin
+title: '📦 eslint-plugin'
+slug: '/api/plugins/@docusaurus/eslint-plugin'
```

Comment:
This shouldn't be in "plugins" because it's not a Docusaurus plugin. Let's put it in a top-level item of itself (or a "Miscellaneous" category where we can put other infrastructure as well)

Reply:
Okay. I assume by top-level you mean a separate entry in the sidebar. So a click on it would:
1. Navigate to `eslint-plugin.md` (`/docs/api/eslint`) which will contain all sections except `## Supported Rules` since that will be split into individual pages, and
2. Reveal the rule pages as dropdown children (one entry for each rule like https://typescript-eslint.io/rules/, e.g. `/docs/api/eslint/no-untranslated-text`).

Do you have a preference for the name ? Could be "ESLint", "ESLint Plugin", "Miscellaneous", ...

Reply:
Done in 9f04ae1.